boards/shields/nrf7002ek_nrf7000/Kconfig.shield

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+DT_COMPAT_NORDIC_NRF700X_QSPI := nordic,nrf700x-qspi
+DT_COMPAT_NORDIC_NRF700X_SPI := nordic,nrf700x-spi
+
+config SHIELD_NRF7002EK_NRF7000
+	def_bool $(shields_list_contains,nrf7002ek_nrf7000)
+	depends on !WPA_SUPP
+
+config NRF7000_ON_QSPI
+	def_bool $(dt_compat_enabled,$(DT_COMPAT_NORDIC_NRF700X_QSPI))
+	depends on !WPA_SUPP && SHIELD_NRF7002EK_NRF7000
+
+config NRF7000_ON_SPI
+	def_bool $(dt_compat_enabled,$(DT_COMPAT_NORDIC_NRF700X_SPI))
+	depends on !WPA_SUPP && SHIELD_NRF7002EK_NRF7000
+
+config BOARD_NRF7000
```

Comment:
Why not use the `SHIELD_NRF7002EK_NRF7000` directly? Also this doesn't conform to the other BOARD_* defines in NRF.

Reply:
BOARD_NRF7000 is to facilitate board-specific checks irrespective of it being a DK or an EK. Thats why SHIELD  config is being translated into a board config.